python/ray/worker.py

```diff
@@ -1914,6 +1914,13 @@ def remote(*args, **kwargs):
 
   def make_remote_decorator(num_return_vals, num_cpus, num_gpus, func_id=None):
     def remote_decorator(func_or_class):
+      try:
+        # Handle the Numba case: if this is a Numba-compiled function, get the underlying Python function
+        func_or_class = getattr(func_or_class, func_or_class.__numba__)
```

Comment:
Nah you're right, I was only trying to fix the error in the other issue. Didn't even cross my mind that we might need to actually jit it afterward haha.